airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4368,6 +4399,11 @@ void setup(void) {
 	createLoggerConfigs();
 	debug_outln_info(F("\nChipId: "), esp_chipid);
 
+	if (cfg::sd_read)
+	{
+		init_SD;
```

Comment:
@Warukira is this a function call? If so are we calling it properly?

Reply:
Missed this one, thanks, fixing it.